device/Microsoft.Azure.Devices.Client/DeviceClient.cs

```diff
@@ -212,6 +232,8 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
         /// </summary>
         Object twinPatchCallbackContext = null;
 
+        internal int CurrentMessageCount = 0;
```

Comment:
Why does this need to be internal instead of private?

Reply:
Oh, you are right! This member can be private.